### PR TITLE
Fix linter issues

### PR DESCRIPTION
**Pull request by Bolodya1997**
## Description
Fix general linter issues for #979

## Motivation and Context
1. Fix general linter issues for #979
2. Issue #1124

## How Has This Been Tested?

- [x] Covered by existing integration testing
- [ ] Added integration testing to cover
- [x] Tested locally
- [ ] Have not tested

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -103,5 +104,11 @@ issues:
     - path: _test\.go
       linters:
         - golint
+    - path: localconnectionmonitor/local_connection_event\.go
```
Comment (denis-tingajkin) [inline .golangci.yml]: can we fix duplicaiton?

Comment (Bolodya1997) [reply .golangci.yml]: No, because there is a need to cast map from abstract type to package-special and the same for enum values.

**Comment by haiodo** [comment]
I’m not like to rename packages with underscores to this long lines, could we try disable linter for this cases?

**Comment by Bolodya1997** [comment]
> I’m not like to rename packages with underscores to this long lines, could we try disable linter for this cases?

@haiodo I have renamed this:
```
- monitor
-- localconnectionmonitor
-- remoteconnectionmonitor
-- crossconnectmonitor
```
to that:
```
- monitor
-- local
-- remote
-- crossconnect
```
so I believe there is no more problem with linter for this case

**Pull request merged**